lib/monad.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
```

Comment:
I only want to show that those functions don't depend on `api`. I will rewrite it.